.github/workflows/conan-package-create-wasm.yml

```diff
@@ -0,0 +1,113 @@
+name: Conan package create Wasm
+
+on:
+  workflow_call:
+    inputs:
+      recipe_id_full:
+        required: true
+        type: string
+
+      conan_internal:
+        required: false
+        default: false
+        type: boolean
+
+      conan_extra_args:
+        required: false
+        default: ""
+        type: string
+
+      conan_recipe_root:
+        required: false
+        default: "."
+        type: string
+
+permissions:
+  contents: read
+
+env:
+  CONAN_LOGIN_USERNAME: ${{ secrets.CONAN_USER }}
+  CONAN_PASSWORD: ${{ secrets.CONAN_PASS }}
+  SENTRY_TOKEN: ${{ secrets.CURAENGINE_SENTRY_TOKEN }}
+
+jobs:
+  package-create:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        if: ${{ github.event.pull_request.head.repo.full_name == github.repository }}
+        with:
+          fetch-depth: 1
+          ref: ${{ github.head_ref }}
+
+      - name: Checkout repo PR
+        uses: actions/checkout@v4
+        if: ${{ github.event.pull_request.head.repo.full_name != github.repository }}
+        with:
+          fetch-depth: 1
+          ref: ${{ github.base_ref }}
```

Comment:
I don't have any major issues with this, I just wonder if this can be resolved in the 'ref' field as:

`ref: ${{ github.event.pull_request.head.repo.full_name == github.repository ? github.head_ref : github.base_ref }}` -> I'm not sure actions supports that syntax though, you may have to do some thing like `(<check equal> && a) || b` instead.

Reply:
Maybe not sure, This is also copied from the other workflows. Maybe keep it consistent for now. Might do a quick check somewhere tomorrow and change that pattern throughout multiple workflows if it can be used as such